examples/spe1/bash.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
```

Comment:
Wrapper script due to the `unix_step` currently executing all commands a `python3 cmd <args>`